cmd/faUniq/faUniq_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"os"
+	"testing"
+)
+
+inputFile := test_in.fa
```

Comment:
You'lll need to envelope test_in.fa in quotation marks so golang can interpret this as a string.
Also, these lines should be placed inside the TestFaUniq function in order to be visible to the faUniq function call